miio/airconditioningcompanion.py

```diff
@@ -306,14 +306,29 @@ def learn_stop(self, slot: int=STORAGE_SLOT_ID):
         return self.send("end_ir_learn", [slot])
 
     @command(
-        click.argument("command", type=str),
+        click.argument("model", type=str),
+        click.argument("code", type=str),
         default_output=format_output("Sending the supplied infrared command")
     )
-    def send_ir_code(self, command: str):
+    def send_ir_code(self, model: str, code: str):
```

Comment:
I'm not a fan of working on a hex string. I think it would be better to convert both model and code strings to bytearrays and work on them. Then at the end, convert it back to a hex string.